config/permissions.yml

```diff
@@ -1161,6 +1161,11 @@ feature_new_statement_form:
     label: 'Stell. verf. Formular anzeigen'
     loginRequired: true
     parent: feature_statement
+feature_not_show_notice_comment_release_draft_statement:
```

Comment:
i find the name a little bit confusing, may be something like 'feature_show_notice_draft_statement' is better ?

Reply:
done in [f6a0d1c](https://github.com/demos-europe/demosplan-core/pull/2721/commits/f6a0d1ca0a0f30aa3b1e35ff7eec9763449ccc11)